.changeset/young-seals-repeat.md

```diff
@@ -0,0 +1,5 @@
+---
+'nextjs': minor
+---
+
+Update to add example with explicit provider
```

Comment:
do we need to update changeset for nextjs project in order for the vercel link to actually update? no right?

Reply:
nope we don't need to update this